waveorder/models/inplane_oriented_thick_pol3d.py

```diff
@@ -98,12 +142,9 @@ def apply_inverse_transfer_function(
         *background_corrected_stokes
```

Comment:
Comments on nomenclature are useful here and throughout other models. I had to look up `estimate_adr_from_stokes` to remember that ADR = attenuating, depolarizing, retarder.